File: ch1/jskim/money.java
class Dollar
{
    int amount;

    Dollar(int amount)
    {
         this.amount = amount;
    }

    void times(int multiplier)
    {
         amount *= multiplier;
    }
}


public void testMultiplication()
{
    Dollar five = new Dollar(5);
    
    five.times(2);
    
    assertEquals(10, five.amount);
}

File: ch1/jskim/money.py
import unittest

class Dollar:
    
    amount:int

    def __init__(self, amount:int):
        self.amount = amount

    def times(self, multiplier:int):
        self.amount *= multiplier

class testMultiplication(unittest.TestCase):

    def testMultiplication(self):
        five = Dollar(5)
        five.times(2)

        self.assertEqual(10, five.amount)
    
if __name__ == "__main__":
    unittest.main()